aiida_pseudo/cli/install.py

```diff
@@ -140,3 +140,111 @@ def cmd_install_sssp(version, functional, protocol, traceback):
         family.set_cutoffs(cutoffs)
 
         echo.echo_success(f'installed `{label}` containing {family.count()} pseudo potentials')
+
+
+@cmd_install.command('pseudodojo')
+@options.VERSION(type=click.Choice(['03', '04']), default='04')
+@options.FUNCTIONAL(type=click.Choice(['pbe', 'pbesol', 'lda']), default='pbe')
+@options.REL(type=click.Choice(['sr', 'sr3plus', 'fr']), default='sr')
+@options.PROTOCOL(type=click.Choice(['standard', 'stringent']), default='standard')
+@options.HINT(type=click.Choice(['low', 'normal', 'high', 'none']), default='none')
+@options.PSEUDO_FORMAT(type=click.Choice(['psp8', 'upf', 'psml']), default='psp8')
+@options.TRACEBACK()
+@decorators.with_dbenv()
+def cmd_install_pseudodojo(version, functional, rel, protocol, hint, pseudo_format, traceback):
+    """Install an PseudoDojo configuration.
+
+    The PseudoDojo configuration will be automatically downloaded from pseudo-dojo.org to create a new
+    `PseudoDojoFamily` subclass based on the correct pseudopotential format.
+    """
+    # pylint: disable=too-many-locals,too-many-arguments,too-many-statements
+    import requests
+    from aiida.common.files import md5_file
+    from aiida.orm import Group, QueryBuilder
+    from aiida_pseudo import __version__
+    from aiida_pseudo.groups.family import (
+        PseudoDojoConfiguration, PseudoDojoFamily, PseudoDojoPsmlFamily, PseudoDojoPsp8Family, PseudoDojoUpfFamily
+    )
+
+    from .utils import attempt, create_family_from_archive
+
+    pseudo_format_families = {'psp8': PseudoDojoPsp8Family, 'psml': PseudoDojoPsmlFamily, 'upf': PseudoDojoUpfFamily}
+
+    metadata_urls = {
+        'nc-sr-04_pbe_standard': 'ONCVPSP-PBE-PDv0.4/standard.djson',
+        'nc-sr-04_pbe_stringent': 'ONCVPSP-PBE-PDv0.4/stringent.djson',
+        'nc-sr-04_pbesol_standard': 'ONCVPSP-PBEsol-PDv0.4/standard.djson',
+        'nc-sr-04_pbesol_stringent': 'ONCVPSP-PBEsol-PDv0.4/stringent.djson',
+        # 'nc-fr-04_pbe_standard': 'ONCVPSP-PBE-FR-PDv0.4/standard.djson',  # md5 does not match
```

Comment:
What will happen if this config is chosen by a user. Will the code except? I think on 221 it will raise a `KeyError`.

Reply:
No, the corresponding PseudoDojoConfigurations are also commented, so you will get the `ValueError` in line 54 of `pseudodojo.py` (`pseudo_dojo.py` after I change the naming).